### PR TITLE
Front end 1

### PR DESCRIPTION
**Pull request by derekmhewitt**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -23,6 +23,13 @@ var/
 *.egg-info/
 .installed.cfg
 *.egg
+<<<<<<< HEAD
```
Comment (nhuntwalker) [inline .gitignore]: This is a remnant from a merge conflict. You should probably remove this from your gitignore.

File: imagersite/static/index.html [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="description" content="">
+    <meta name="author" content="">
+
+    <title>Portfolio Item - Start Bootstrap Template</title>
+
+    <!-- Bootstrap Core CSS -->
+    <link href="css/bootstrap.min.css" rel="stylesheet">
```
Comment (nhuntwalker) [inline imagersite/static/index.html]: This and the next `link` tag should be brought in using `{% static %}`

File: imagersite/templates/imagersite/base.html [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+                        <a href="#">About</a>
+                    </li>
+                    <li>
+                        <a href="#">Services</a>
+                    </li>
+                    <li>
+                        <a href="#">Contact</a>
+                    </li>
+                    <li>
+                        <a href="/register/login/">Login</a>
```
Comment (nhuntwalker) [inline imagersite/templates/imagersite/base.html]: This URL should be generated with `{% url %}`, not manually written.

File: imagersite/templates/imagersite/home.html [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+            <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam viverra euismod odio, gravida pellentesque urna varius vitae. Sed dui lorem, adipiscing in adipiscing et, interdum nec metus. Mauris ultricies, justo eu convallis placerat, felis enim.</p>
+            <h3>Project Details</h3>
+            <ul>
+                <li>Lorem Ipsum</li>
+                <li>Dolor Sit Amet</li>
+                <li>Consectetur</li>
+                <li>Adipiscing Elit</li>
+            </ul>
+        </div>
+
```
Comment (nhuntwalker) [inline imagersite/templates/imagersite/home.html:28]: Where's the call to action to register?

File: imagersite/templates/registration/registration_form.html [hunk 5]
```diff
@@ -0,0 +1,14 @@
+{% extends "imagersite/layout.html" %}
+{% block content %}
+    <p>Register for a new account here.</p>
+    <form>
```
Comment (nhuntwalker) [inline imagersite/templates/registration/registration_form.html]: Where's your form?

File: imagersite/urls.py [hunk 6]
```diff
@@ -0,0 +1,43 @@
...
+    url(r'^admin/', include(admin.site.urls)),
+    url(r'^accounts/', include('registration.backends.hmac.urls')),  # Note 1
+    url(r'^profile/', include('imager_profile.urls')),
+    url(r'^images/', include('imager_images.urls')),
+]
+
+if settings.DEBUG:
+    urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
+
+"""
```
Comment (nhuntwalker) [inline imagersite/urls.py]: You should remove this or put it into a comment, not a free-floating string

File: imagersite/views.py [hunk 7]
```diff
@@ -0,0 +1,12 @@
+from django.shortcuts import render
+from django.http import HttpResponse
+from django.template import loader
+
+
+def index_view(request):
+    template = loader.get_template('imagersite/home.html')
+    # import pdb; pdb.set_trace();
+    response_body = template.render()
+    return HttpResponse(response_body)
```
Comment (nhuntwalker) [inline imagersite/views.py:10]: So no random image?

File: .gitignore (hunk 1 above)
Comment (derekmhewitt) [reply .gitignore]: Done

File: imagersite/static/index.html (hunk 2 above)
Comment (derekmhewitt) [reply imagersite/static/index.html]: This file isn't actually being used, it was just a basis for our templates and I kept it as a reference.  I'll go ahead and remove it now.

File: imagersite/templates/imagersite/base.html (hunk 3 above)
Comment (derekmhewitt) [reply imagersite/templates/imagersite/base.html]: Same thing, I'll remove this file as well.

File: imagersite/templates/imagersite/home.html (hunk 4 above)
Comment (derekmhewitt) [reply imagersite/templates/imagersite/home.html:28]: Added.

File: imagersite/urls.py (hunk 6 above)
Comment (derekmhewitt) [reply imagersite/urls.py]: I cleaned up this entire file and removed all the comments and imports we're no longer using.

File: imagersite/templates/registration/registration_form.html (hunk 5 above)
Comment (derekmhewitt) [reply imagersite/templates/registration/registration_form.html]: Added.

**Pull request merged**